schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_entry.json

```diff
@@ -10289,6 +10289,894 @@
          "minItems": 1,
          "uniqueItems": true
       },
+      "pdbx_vrpt_summary": {
+         "type": "object",
+         "properties": {
+            "EMDB_deposition_date": {
+               "type": "string",
+               "format": "date",
+               "description": "Date in yyyy-mm-dd format when map was deposited to the EMDB.",
+               "rcsb_description": [
+                  {
+                     "text": "Date in yyyy-mm-dd format when map was deposited to the EMDB.",
+                     "context": "dictionary"
+                  }
+               ]
+            },
+            "PDB_deposition_date": {
```

Comment:
Sorry for back and forth, we might want to suppress these too as they seem to duplicate data from model files

Reply:
@valasatava. Yes we can suppress `PDB_deposition_date`. How about `PDB_revision_date` and `PDB_revision_number` in the same object?

Reply:
I'd say any "PDB_" attribute since those seem to be duplicating data from model file